accepted/2020/workloads/workloads.md

```diff
@@ -0,0 +1,92 @@
+
+This means that optional components will not become NuGet packages that require package references in project files, or that are acquired as part of package restore. Instead, a new system will be developed that is an evolution of the SDK composition system, that evolves from a physically monolithic SDK to a logically monolithic one. This approach will enable us to deliver SDK UX with .NET 5 that is similar in spirit to the .NET Core 3.1 SDK, with the acknowledgment that there is some regression in functionality (you don't get the kitchen and its sink by default).
+
+Since the first .NET Core release, we have delivered various types of "packs" as specialized packages on NuGet feeds ([example from 1.x and 2.x era](https://www.nuget.org/packages/runtime.win-x64.Microsoft.NETCore.App/); [example from 3.x+ era (unlisted)](https://www.nuget.org/packages/Microsoft.NETCore.App.Runtime.linux-x64)). We use NuGet for this purpose because many tools and services understand NuGet packages, it includes a concept of feeds (which can be private or public), and offers a global CDN (in the case of NuGet.org). These packs, however, are not intended to be directly referenced by project files, but are known to the .NET Core SDK as an implementation detail. We intend workloads to follow a similar model, with the difference that the .NET SDK will not have detailed built-in knowledge of packs, and that workloads will be separately acquirable and updatable from the SDK.
+
+### Composition
+
+Workloads will be composed and distributed as a set of packs. Packs are just a compressed set of files, like the [ASP.NET Core runtime]((https://www.nuget.org/packages/Microsoft.AspNetCore.App.Runtime.linux-arm64)). They are not discoverable on their own, nor do they describe the other packs that may work with or require or the various scenarios in which they can be used. Today, there are many hard-coded [versions and packages described in MSBuild targets](https://github.com/dotnet/core-sdk/blob/edbdf82041abff369ed64e5d0d6fd590e75b5328/src/redist/targets/GenerateBundledVersions.targets) in the SDK that satisfy this need. Moving forward, we will externalize this compositional information into a set of workload manifest files. Each workload will include a manifest (likely distributed as its own small pack), and use it to describe the set of versioned components that it offers. MSBuild and potentially other tools will be updated to discover and read these manifests, to the end of honoring them in the build and any other relevant operations.
```

Comment:
Split this paragraph, and in the first paragraph add that a variety of things will be included in packs (since you mentioned runtimes) including templates and design time components.

Reply:
The term "versioned components" is confusing near the end. Either replace that work with "pack" or add a sentence that further clarifies that packs contained versioned components. Although I think I'd prefer to see the whole system defined with no use of the word "version" and then a section on how versioning works.

Reply:
I think we want to be cautious flipping between the logical and physical view. Consider the phrase "Each workload will include.." Maybe "Each logical workload will have an associated manifest..."

Reply:
OK. I think I fixed those things.